### PR TITLE
Update 6.5.4

### PR DESCRIPTION
**Pull request by Jrice1317**
- Updated version and hash
- Alphabetized run section
- Updated pinnings
- Added install_reqs
- Updated urls
- Added license and doc_source urls

Jira: https://anaconda.atlassian.net/browse/PKG-155
https://github.com/jupyter/nbconvert/tree/main

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

**Review by ELundby45 — APPROVED** [review]
LGTM. I'll just add a reminder to remove the abs.yaml before deploying.

File: abs.yaml
Comment (katietz) [inline abs.yaml]: please remove this channel stuff

File: recipe/meta.yaml
Comment (katietz) [inline recipe/meta.yaml]: there is a license_file in package.  No need for the license_url.  It is superflous, even problematic mid/long term

File: recipe/meta.yaml
Comment (katietz) [inline recipe/meta.yaml]: doc_source_url is superflous.  there is already a doc_url

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml
Comment (katietz) [reply recipe/meta.yaml]: done it for you.  Next time please adjust before asking me for a second review

File: recipe/meta.yaml
Comment (katietz) [reply recipe/meta.yaml]: done for you ...

File: abs.yaml
Comment (katietz) [reply abs.yaml]: done for you

**Comment by katietz** [comment]
please upload first tinycss2 before building this package!

**Comment by katietz** [comment]
after uploading its dependencies, it can be build without private channel information abs.yaml, as you see now

**Review by katietz — APPROVED** [review]
after all issues are resolved, final build was successful without private channel, the PR is ready for approval. and it looks good to me now.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**
